.github/ISSUE_TEMPLATE/documentation.yaml

```diff
@@ -0,0 +1,40 @@
+name: 📚Documentation
```

Comment:
```suggestion
name: 📚 Documentation
```

there is a space between emoji and 'Documentation'

Reply:
@chaso1495 This one should be updated :) Check this comment, please 😸

Reply:
@yeonns I checked and fixed it :)